### ISSUE TITLE
Unidiomatic representation of namespaced maps

### ISSUE THREAD

**Issue by weavejester** [greglook/puget#51]
Puget represents namespaced maps as:

    #:foo {:x 1, :y 2}

But Clojure's printer omits the space between the namespace and the map:

    #:foo{:x 1, :y 2}

Puget should probably follow the convention that's used by the Clojure printer.

**Comment by greglook** [comment]
Yeah, this is mainly an artifact of using the `:line` separator in `fipp`, which becomes a single space when the form fully fits onto one line:
https://github.com/greglook/puget/blob/f458a0dd704e37fde348aebed7cd8ed5248edff0/src/puget/printer.clj#L586

The behavior I wanted from that is so that really long namespaces get line broken like this:
```clojure
#:acme.foo.bar.baz
{:a 123
 :b true}
```
...rather than like this:
```clojure
#:acme.foo.bar.baz{:a 123
                   :b true}
```
I'll see if there's an easy way to get fipp to omit the space in the one-line case.
